src/schematools/models.py

```diff
@@ -10,6 +10,7 @@
 class Dataset(Base):
     __tablename__ = "ams_schema_dataset"
     id = Column(String, primary_key=True)
+    path = Column(String)
```

Comment:
Here there is no length specified. This effectively boils down to the `TEXT` PostgreSQL type. So we now have two different definition for `path`:
- Django -> 250 -> `VARCHAR(250)
- SQLAlchemy -> unspecified -> `VARCHAR` -> `TEXT`

The SQLAlchemy one seems to match what I remember from the spec (when I could found it in the spec)